### PR TITLE
conference page project

### PR DESCRIPTION
**Pull request by chuanqizhu0820**
## summary
The following items have been accomplished: 
* The pages should look almost identical to the original design. 
* use Bootstrap for styling
* build the home page and the about page.
* Each of these pages has mobile and desktop versions
* The home page has a link to the about page.
* The logo links to the home page.
* hamburger button => the mobile menu 
* mobile menu has a close (X) button that closes the menu.
* "Featured speakers" are created dynamically in JavaScript.
* use a JavaScript variable with the data about the speakers
* give credit to the author in the README

Here is the loom video to show you around my website! 
https://www.loom.com/share/0198da3f0c4240778c7a11fc3195fac2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,56 +12,56 @@
...
+                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                        data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false"
+                        aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+                    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                        <ul class="navbar-nav">
+                            <li class="nav-item "><a href="http://127.0.0.1:5500"><img id="logo"
+                                        src="img/icons/logo.png" alt="logo"></a>
+                            </li>
```
Comment (george-shammar) [inline index.html:38]: - [ ] There should be links from `Home` page to `About` page and the opposite too. The link back to the `Home` Page in the live version does not work. An error page is being displayed. Please, let's fix it. Hint: `http://127.0.0.1:5500` in your `a` tag is the source of the error.

File: index.html [hunk 2]
```diff
@@ -12,56 +12,56 @@
...
+            </div>
+            <nav class="navbar navbar-expand-md navbar-light">
+                <div class="container-fluid">
+                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse"
+                        data-bs-target="#navbarNavDropdown" aria-controls="navbarNavDropdown" aria-expanded="false"
+                        aria-label="Toggle navigation">
+                        <span class="navbar-toggler-icon"></span>
+                    </button>
+                    <div class="collapse navbar-collapse" id="navbarNavDropdown">
+                        <ul class="navbar-nav">
```
Comment (george-shammar) [inline index.html:35]: - [ ] Your HTML should use semantic tags. Articles should use `article`, the nav and footer should use `nav` and `footer`.

**Review by george-shammar — CHANGES_REQUESTED** [review]
Hi @chuanqizhu0820 , 

Good job so far!

:heavy_check_mark: You have done an awesome job with this project. YOur design looks almost exactly as the design template. Great job. :+1: 


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chuanqizhu0820** [comment]
Here is the new loom video with code highlight!
https://loom.com/share/06a64564b841421cb03df7d623df58fa

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @chuanqizhu0820 , 

Great presentation :+1: 
I can see that you have made all of the requested changes :clap: 
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**